edflow/hooks/logging_hooks/minimal_logging_hook.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, paths, interval, root_path, name=None):
         for path in self.paths:
             os.makedirs(os.path.join(self.root, path), exist_ok=True)
             self.loggers[path] = get_logger(path)
-        self.handlers = {"images": [self.log_images], "scalars": [self.log_scalars]}
+        self.handlers = {"images": [self.log_images], "scalars": [self.log_scalars], "figures": [self.log_figures]}
```

Comment:
Nice! That is a nice functionality add-on!

Reply:
Thanks!

Reply:
Updated the CHANGELOG please approve